### PR TITLE
Updated to the replacement of Device.Addressable with Device Protocols

### LINKED ISSUE #30: Update due to Device changes and Schedule* deprecation
The example configuration file needs to be updated to the replacement of `Device.Addressable` with `Device.Protocols`, and the deprecation of Schedule* objects by device-sdk-go.

The same `Device` **configuration.toml** and example driver changes that were made to device-sdk-go in commit [fc71221](https://github.com/edgexfoundry/device-sdk-go/commit/fc712212962e7a52da8ff0b7d6a8734cc5af8bdd) need to be applied. 

Likewise the Schedule* **configuration.toml** changes (see device-sdk-go commit [2db9552](https://github.com/edgexfoundry/device-sdk-go/commit/2db9552350484545cc2f0750a9a33b6f572fc05c) also need to be applied.

### PR DESCRIPTION
**Pull request by weichou1229**
Fix #30 
Updated to the replacement of Device.Addressable with Device Protocols, and the deprecation of Schedule* objects by device-sdk-go

### CODE REVIEW

File: cmd/res/example/configuration.toml [hunk 1]
```diff
@@ -58,53 +58,18 @@ Level = "DEBUG"
...
-[[ScheduleEvents]]
-Name = "Read Values"
-Schedule = "5sec-schedule"
-  [ScheduleEvents.Addressable]
-  HTTPMethod = "GET"
-  Path = "/api/v1/device/name/Modbus TCP test device/Values"
-
+  [DeviceList.Protocols]
+    [DeviceList.Protocols.modbus-rtu]
+       Address = "/tmp/slave,19200,8,1,0"
```
Comment (cloudxxx8) [inline cmd/res/example/configuration.toml]: there is no need to separate several value in Address anymore
please split it to different protocol properties

Comment (weichou1229) [reply cmd/res/example/configuration.toml]: Got it. Modified.

**Comment by cloudxxx8** [comment]
Since the SDK API is changed to use ProtocolProperties, could you please rework on this?
thanks

**Comment by weichou1229** [comment]
I have modified the func according to the change.

**Comment by weichou1229** [comment]
I have modified the func according to the change.

**Pull request merged**